src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.cs

```diff
@@ -8,17 +8,18 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Is this using used?

Reply:
Yes, `NotNullWhen` on `tryDecodeExtensionType`